tests/test_linters.py

```
@@ -0,0 +1,26 @@
+import subprocess
+
+
+def test_pylint_runs_clean() -> None:
+    result = subprocess.run(
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)